test/cucumber/steps/cli.js

```diff
@@ -12,11 +12,15 @@ module.exports = function() {
     'I validate the message using the following Gavel command:',
     async function(command) {
       this.commands.push(command);
-      this.status = await this.executeCommands(this.commands);
+      this.exitCode = await this.executeCommands(this.commands);
     }
   );
 
-  this.Then(/^exit status is (\d+)$/, function(expectedStatus) {
-    assert.equal(this.status, expectedStatus, 'Process statuses do not match');
+  this.Then(/^exit status is (\d+)$/, function(expectedExitCode) {
```

Comment:
Would be nice to change the wording also in the spec for consistency, but let's not bother with this now.